tx_utils.go

```diff
@@ -0,0 +1,31 @@
+package lndclient
+
+import (
+	"bytes"
+
+	"github.com/btcsuite/btcd/wire"
+)
+
+// encodeTx encodes a tx to raw bytes.
+func encodeTx(tx *wire.MsgTx) ([]byte, error) {
```

Comment:
Might as well export these as we're building a library anyway.

Reply:
I think this is functionality to belongs more in `github.com/btcsuite/btcd/wire`